scripts/zones/La_Theine_Plateau/mobs/Bloodtear_Baldurf.lua

```diff
@@ -19,4 +30,28 @@ end;
 
 function onMobDeath(mob, player, isKiller)
     player:addTitle(THE_HORNSPLITTER);
-end;
\ No newline at end of file
+end;
+
+-----------------------------------
+-- onMobDespawn
+-----------------------------------
+
+function onMobDespawn(mob)
+    local mobID = mob:getID();
+    local chanceForLambert = 0;
+
+    if (GetServerVariable("[POP]Lumbering_Lambert") <= os.time(t)) then
+        local chanceForLambert = math.random(1,100);
```

Comment:
Already in the scope of the function and your if block already has that one. There is no "which one" - there's only one, not two of it. :)